desktop_server/src/com/mygdx/game/server/network/EndTurnService.java

```diff
@@ -0,0 +1,68 @@
+package com.mygdx.game.server.network;
+
+import com.artemis.World;
+import com.mygdx.game.core.ecs.component.PlayerToken;
+import com.mygdx.game.server.ecs.entityfactory.ComponentFactory;
+import com.mygdx.game.server.model.Client;
+import com.mygdx.game.server.model.GameRoom;
+import lombok.NonNull;
+import lombok.extern.java.Log;
+
+import javax.inject.Inject;
+import java.util.List;
+
+@Log
+public class EndTurnService {
+
+  private final int playerTokenComponentId;
+  private final PlayerToken playerToken;
+  private final GameRoom gameRoom;
+  private final GameRoomSyncer gameRoomSyncer;
+
+  @Inject
+  EndTurnService(
+      @NonNull World world,
+      @NonNull ComponentFactory componentFactory,
+      @NonNull GameRoom gameRoom,
+      @NonNull GameRoomSyncer gameRoomSyncer
+  ) {
+    this.playerTokenComponentId = componentFactory.createEntityId();
```

Comment:
Ten serwis specjalnie nie jest singletonem? Bo teraz za każdym razem ten componentFactory.createEntityId() tworzy faktycznie inne Id gdy jest gdzieś inejctowany, ale nie wiem czy to zamierzone

Reply:
Zapomniałem dać